src/Core/src/Platform/Android/ButtonExtensions.cs

```diff
@@ -96,7 +96,7 @@ internal static void UpdateBorderDrawable(this MaterialButton platformView, IBut
 				if (button.StrokeThickness > 0)
 					mauiDrawable.SetBorderWidth(button.StrokeThickness);
```

Comment:
Does this also need to be changed? Sure, this PR is not meant for both because of the issue, but maybe both can be fixed at the same time? Not sure if you observed strange things with borders?

If you would like to keep these 2 things separate, the that is good. I'll just create an issue so we don't miss it.

Reply:
One problem at a time, man.

Reply:
I noticed this as well, but didn't find an issue related to it. So I am unsure if strange things happen here, to be checked.